release/cloud_build.template.json

```diff
@@ -8,7 +8,7 @@
 	}
       ],
       "name": "gcr.io/istio-testing/istio-builder:0.5.7",
-      "args": [ "-c", "repo init -u \"$_MFEST_URL\" -m \"$_MFEST_FILE\" -b \"$_MFEST_VER\" && repo sync && cp .repo/manifest.xml /output/ && echo \"$_MFEST_VER\" > /output/green_build_sha.txt" ],
+      "args": [ "-c", "git clone https://github.com/istio/istio -b \"$_BRANCH\" --depth 1 && istio/release/istio_checkout_code.sh -b \"$_BRANCH\" -r \"$_GCS_RELEASE_TOOLS_PATH\" " ],
```

Comment:
1, format json list to make these more readable.

2. move/refactor CB specific files to use either a consistent prefix or in a sub-dir.